src/TodoForm.jsx

```diff
@@ -1,9 +1,23 @@
-function TodoForm() {
+import { useRef } from 'react';
+
+function TodoForm({ onAddTodo }) {
+  const todoTitleInput = useRef("");
+
+  function handleAddTodo(event) {
+    event.preventDefault();
+    const title = event.target.title.value;
+    event.target.title.value = "";
+    onAddTodo({ 'id': Date.now(), 'title': title });
+    todoTitleInput.current.focus();
+  }
+
   return (
     <>
-      <label htmlFor="todoTitle">Todo &nbsp;</label>
-      <input id="todoTitle"></input>
-      <button>Add Todo</button>
+      <form onSubmit={handleAddTodo}>
+        <label htmlFor="todoTitle">Todo &nbsp;</label>
+        <input id="todoTitle" ref={todoTitleInput} name="title"></input>
+        <button>Add Todo</button>
```

Comment:
By default a button type is `type="submit"` but it's still good practice to explicitly state that.